python/setup.py

```diff
@@ -20,6 +29,10 @@
 
 # NOTE: The lists below must be kept in sync with ray/BUILD.bazel.
 
+pyarrow_url = "https://s3-us-west-2.amazonaws.com/arrow-wheels/3a11193d9530fe8ec7fdb98057f853b708f6f6ae/index.html"
```

Comment:
please make global constants `ALL_CAPS_SNAKE_CASE`

Reply:
@edoakes the other variables below aren't like this though?

Reply:
ah, they should be :) but I guess this is fine because it's `setup.py` and not a normal python program...

Reply:
Yeah I think it's the setup.py convention.